code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -693,13 +723,17 @@
 	var/old_heat_capacity = air.heat_capacity()
 	var/heat_efficency = min(temperature * 0.005, cached_gases[/datum/gas/pluoxium][MOLES] * INVERSE(0.2), cached_gases[/datum/gas/hydrogen][MOLES] * INVERSE(2))
 	var/energy_used = heat_efficency * 650
+
 	ASSERT_GAS(/datum/gas/proto_nitrate, air)
 	if ((cached_gases[/datum/gas/pluoxium][MOLES] - heat_efficency * 0.2 < 0 ) || (cached_gases[/datum/gas/hydrogen][MOLES] - heat_efficency * 2 < 0)) //Shouldn't produce gas from nothing.
 		return NO_REACTION
+
 	cached_gases[/datum/gas/hydrogen][MOLES] -= heat_efficency * 2
 	cached_gases[/datum/gas/pluoxium][MOLES] -= heat_efficency * 0.2
 	cached_gases[/datum/gas/proto_nitrate][MOLES] += heat_efficency * 2.2
 
+	air.reaction_results[/datum/gas_reaction/proto_nitrate_formation] = heat_efficency * 2.2
```

Comment:
could you make this a macro? we're never going to want to set a reaction result that doesn't come from our type, will we?

Reply:
12f6d0751abd282546f1da232657411c7f3c8f75

Reply:
actually i think we can even remove the gas_mixture there. second.

Reply:
If it's going to be that fine tuned, it should just be defined in the reactions file, and undeffed at the end